package.json

```diff
@@ -36,14 +36,16 @@
 		"onCommand:cosmosDB.createMongoDatabase",
 		"onCommand:cosmosDB.createDocDBDatabase",
 		"onCommand:cosmosDB.createDocDBCollection",
+		"onCommand:cosmosDB.createDocDBDocument",
 		"onCommand:cosmosDB.connectMongoDB",
 		"onCommand:cosmosDB.deleteMongoDB",
 		"onCommand:cosmosDB.deleteDocDBDatabase",
 		"onCommand:cosmosDB.deleteDocDBCollection",
+		"onCommand:cosmosDB.deleteDocDBDocument",
 		"onCommand:cosmosDB.newMongoScrapbook",
-		"onCommand:cosmosDB.updateMongoDocuments",
+		"onCommand:cosmosDB.updateDocuments",
```

Comment:
Couldn't come up with a clear name for this. 
We are trying to consolidate two slightly different operations: 
1. Mongo - Update **collection**
2. DocDB - Update **document**

Any feedback?